pkg/updater/config.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file defines the user configuration for the updater.
+
+package updater
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+// userConfig is the user configuration for the updater.
+type userConfig struct {
+	// path is the path to this user configuration
+	path string
+
+	// AlwaysUsePrereleases instructs the updater to always consider prereleases
+	AlwaysUsePrereleases bool `yaml:"alwaysUsePrereleases"`
+}
+
+// readConfig reads the user's configuration from a well-known path
+func readConfig(repo string) (userConfig, error) {
```

Comment:
return a pointer to `userConfig`? the type uses pointer semantics anyways on its receiver functions